File: julia/covar_series_data.jl
using Random
using Distributions
Random.seed!(1)

n = 10
F = randn((10, 10))
Sigma = F' * F
T = 10
a = [0.2 0.1 0.2 0.4 0.8 1.0 1.0 0.8 0.7 0.8]
x = zeros((n, T))
y = zeros((n, T))
for t in 1:T
    dist = MvNormal(zeros(n), a[t]*Sigma)
    x[:, t] = rand(dist)
    y[:, t] = rand(dist)
end

File: python/covar_series_data.py
import numpy as np
np.random.seed(1)

n = 10
F = np.random.randn(10, 10)
Sigma = F.T@F
T = 10
a = np.array([0.2, 0.1, 0.2, 0.4, 0.8, 1.0, 1.0, 0.8, 0.7, 0.8])
x = np.zeros((n, T))
y = np.zeros((n, T))
for t in range(T):
    x[:, t] = np.random.multivariate_normal(np.zeros(n), a[t]*Sigma)
    y[:, t] = np.random.multivariate_normal(np.zeros(n), a[t]*Sigma)
